src/unix/notbsd/linux/mod.rs

```diff
@@ -1616,6 +1616,9 @@ extern {
                            len: ::off_t) -> ::c_int;
     pub fn readahead(fd: ::c_int, offset: ::off64_t,
                      count: ::size_t) -> ::ssize_t;
+    pub fn renameat2(olddirfd: ::c_int, oldpath: *const ::c_char,
+                     newdirfd: ::c_int, newpath: *const ::c_char,
+                     flags: ::c_int) -> ::c_int;
```

Comment:
according to the linux [docs](http://man7.org/linux/man-pages/man2/rename.2.html#NOTES), this should be `c_uint`, but `RENAME_EXCHANGE` which can be passed here is already defined in `libc` as `c_int`. [docs](https://docs.rs/libc/0.2.42/libc/constant.RENAME_EXCHANGE.html)